### ISSUE TITLE
A11y shrine: add missing headers

### ISSUE THREAD

**Issue by felixzapata** [PolymerElements/app-layout#360]
### Description

There is a lack of headers elements in the items and in the feature item.
### Expected outcome

The page should have some headers to help the users to navigate between them.
### Actual outcome

There are not headers in the feature item neither in the item inside the list.
